GUI/PyQt/config/DatabaseInfo.py

```diff
@@ -3,7 +3,7 @@
 import csv
```

Comment:
Please ensure that all modifications to in GUI/PyQt/config/DatabaseInfo.py still allow correct processing with /DatabaseInfo.py --> I will delete DatabaseInfo.py in a next commit and link then to these files!

Reply:
Same applies to GUI/PyQt/config/MRData.py